inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/source/pulsar/PulsarSourceOperator.java

```diff
@@ -126,7 +126,12 @@ public Map<String, List<StreamSource>> getSourcesMap(InlongGroupInfo groupInfo,
             pulsarSource.setTopic(streamInfo.getMqResource());
             pulsarSource.setAdminUrl(adminUrl);
             pulsarSource.setServiceUrl(serviceUrl);
-            pulsarSource.setInlongComponent(true);
+
+            // CSV: InLong message type whose message body is raw CSV
+            // Raw-CSV: messages are separated by a specific separator
+            // When using RAW_CSV format, not an inlong component
+            boolean isInlongComponent = !"RAW_CSV".equals(streamInfo.getDataType());
```

Comment:
It is recommended to move constants to `InlongConstants`, like `InlongConstants.DATA_TYPE_RAW_CSV`